### DISCUSSION TITLE
Add a different Value Help for Source/Destination Accounts

### DISCUSSION THREAD

**Discussion by Roxas2712** [victorbalssa/abacus#19] [Ideas]
Hello :) Would it be possible to have a different valuehelp here that would allow me to enter something new or is doing a „lookup“ within my accounts and making suggestions? Just like firefly is doing it ^^.

i have the problem that I did an import of all my transactions and I’m somehow now not able to use the app anymore because I can’t enter a correct account.  :(  Don’t now if this is maybe also a bug but if not it would be another idea vor an improvement. Find attached a screenshot. I can just scroll to „F“ - then the list ends.

![5EB8106D-0253-41F7-A2C4-1E54B19206BD](https://user-images.githubusercontent.com/41802861/180350213-ab353a0e-394f-4e78-9c38-921fb067094b.jpeg)

**Comment by victorbalssa** [comment]
Thank you, after trying to recreate the same amount of `Accounts` it was due to the `per_page` limit.

It will be fix in the next release #20.

**Reply by Roxas2712** [reply]
Great thank you - will you be then able to add also
new accounts?

**Reply by victorbalssa** [reply]
What do mean by 'new accounts'?

Newly created accounts? 
Why do you need to fetch new accounts?

In my point of view, we don't usually create new bank accounts on a regular basis right?

**Reply by Roxas2712** [reply]
Nope I don’t mean bank accounts. I’m talking about expand accounts. For example: I go to a mall where I’ve never been before and would like to directly, after paying, enter the „new“ mall as destination and my bank account from where I’ve paid, as source. This is what I mean. Hopefully it makes it clearer now^^.

Because - what I’ve seen in firefly is when I enter a new transaction I also can enter a „new“ source/target „account“.

**Reply by victorbalssa** [reply]
Nice, I see it now.

I will create and issue to implement a solution.
#22
